### PR TITLE
chore: update ingest azure cognitive search endpoint

### PR DESCRIPTION
**Pull request by ahmetmeleq**
This PR:
- updates ingest azure cognitive search destination connector test to move into a new service.
- changes response parsing logic in the test.

### CODE REVIEW

File: test_unstructured_ingest/dest/azure-cognitive-search.sh [hunk 1]
```diff
@@ -28,13 +28,13 @@ source "$SCRIPT_DIR"/cleanup.sh
 function cleanup {
   # Index cleanup
   response_code=$(curl -s -o /dev/null -w "%{http_code}" \
-    "https://utic-test-ingest-fixtures.search.windows.net/indexes/$DESTINATION_INDEX?api-version=$API_VERSION" \
+    "https://ingest-test-azure-cognitive-search.search.windows.net/indexes/$DESTINATION_INDEX?api-version=$API_VERSION" \
```
Comment (ryannikolaidis) [inline test_unstructured_ingest/dest/azure-cognitive-search.sh:31]: nit: should we make a variable at the top? (based on re-use)

Comment (ahmetmeleq) [reply test_unstructured_ingest/dest/azure-cognitive-search.sh:31]: Can do, on it

File: test_unstructured_ingest/dest/azure-cognitive-search.sh [hunk 2]
```diff
@@ -51,12 +51,13 @@ trap cleanup EXIT
...
 echo "Creating index $DESTINATION_INDEX"
 response=$(curl -X PUT -s -w "\n%{http_code}" \
-  "https://utic-test-ingest-fixtures.search.windows.net/indexes/$DESTINATION_INDEX?api-version=$API_VERSION" \
+  "https://ingest-test-azure-cognitive-search.search.windows.net/indexes/$DESTINATION_INDEX?api-version=$API_VERSION" \
   --header "api-key: $AZURE_SEARCH_API_KEY" \
   --header 'content-type: application/json' \
   --data "@$SCRIPT_DIR/files/azure_cognitive_index_schema.json")
 response_code=$(tail -n1 <<<"$response") # get the last line
 content=$(sed '$ d' <<<"$response")      # get all but the last line which contains the status code
+response_code="${response_code//[$'\t\r\n ']}" # remove all whitespace to allow integer comparison
```
Comment (ryannikolaidis) [inline test_unstructured_ingest/dest/azure-cognitive-search.sh:60]: was this something we were missing before? noticed in testing?

Comment (ahmetmeleq) [reply test_unstructured_ingest/dest/azure-cognitive-search.sh]: Mine was wrong. Streaming somehow doesn't work for me but this wasn't the solution. Here's the solution I found for my case: [change response parsing, parametrize url](https://github.com/Unstructured-IO/unstructured/pull/2353/commits/96bb37a7fdcadb29a0ceb6a1f5df7420ca119f6e)

**Review by ryannikolaidis — APPROVED** [review]
lgtm!

**Pull request merged**